browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -554,3 +584,212 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
   EXPECT_EQ("third-party-a.com", third_party_values.session_storage);
   EXPECT_EQ("name=third-party-a.com", third_party_values.cookies);
 }
+
```

Comment:
you also need to test shield and chromium site overrides for block all and block 3p based on the discussion we had with @pes10k about that behavior

Reply:
also you don't have any tests for BlockAll and I'm fairly certain they won't work correctly with these changes

Reply:
So all storage should not be  accessible when BlockAll cookies, no matter ephemeral flags on or off?
OK, will do.